dango/types/src/oracle/price.rs

```diff
@@ -55,3 +82,28 @@ impl TryFrom<PriceFeed> for PrecisionlessPrice {
         })
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use grug::NumberConst;
+
+    use super::*;
```

Comment:
Combine the `use` statements into a single one:

```rust
use {grug::NumberConst, super::*};
```

and add the separator comment:

```rust
// ----------------------------------- tests -----------------------------------
```